recipes/ctre/all/conandata.yml

```diff
@@ -2,3 +2,33 @@ sources:
   "2.8.2":
     url: https://github.com/hanickadot/compile-time-regular-expressions/archive/v2.8.2.tar.gz
     sha256: f89494f52ec31e5854fff3d2c5825474201476636c5d82a9365dad5188396314 
+  "2.8.3":
+    url: https://github.com/hanickadot/compile-time-regular-expressions/archive/v2.8.3.tar.gz
+    sha256: 5833a9d0fbce39ee39bd6e29df2f7fcafc82e41c373e8675ed0774bcf76fdc7a 
+  "2.8.4":
+    url: https://github.com/hanickadot/compile-time-regular-expressions/archive/v2.8.4.tar.gz
+    sha256: 99b981857f1b66cab5e71161ae74deca268ed39a96ec6507def92d4f445cadd6 
+  "2.9.0":
+    url: https://github.com/hanickadot/compile-time-regular-expressions/archive/v2.9.0.tar.gz
+    sha256: 1c15f4ad37a4adfc0550f65bec092f34cb4a3fef443cb8e83a303accbf96779c 
+  "2.9.1":
+    url: https://github.com/hanickadot/compile-time-regular-expressions/archive/v2.9.1.tar.gz
+    sha256: 1b52af758af9c8bae8ee0a011dc3076db4c37ffb5c93b3b5098eb9359bf4d1c3 
+  "2.9.2":
+    url: https://github.com/hanickadot/compile-time-regular-expressions/archive/v2.9.2.tar.gz
+    sha256: 763cb6f26e6fc68b4c98a809efbf55aab1a21c7773da8150d41c31baf5f8b711 
+  "2.10":
+    url: https://github.com/hanickadot/compile-time-regular-expressions/archive/v2.10.tar.gz
+    sha256: 2e166b672c5a733b5448bb9fc83db54d92852c39e3e19097b79307cc6a327c31 
+  "3.0":
+    url: https://github.com/hanickadot/compile-time-regular-expressions/archive/v3.0.tar.gz
+    sha256: b1f24383a4c12c68a9da6e61283e02c1e6ff27875af33a8a95ccca72256b4dc9
+  "3.0.1":
+    url: https://github.com/hanickadot/compile-time-regular-expressions/archive/v3.0.1.tar.gz
+    sha256: 0838ce0af9e9e89ffc10854dc98c88b2dd40d1866ceaf0dc6c1663d6c39882cc
+  "3.1":
+    url: https://github.com/hanickadot/compile-time-regular-expressions/archive/v3.1.tar.gz
+    sha256: c209ab0533b56a4fbbc7959e10c91c89fe2c9cb1725b64ed52be272a534a1d83
+  "3.2":
+    url: https://github.com/hanickadot/compile-time-regular-expressions/archive/v3.2.tar.gz
+    sha256: d4e734e3dd501f741424aceb5198f5ebbc35c624335c99f7e202a9ecb31cd3e6
```

Comment:
This is quite excessive. Please only add versions you actually need

Reply:
I've cut out all but the latest of each major/minor.